src/main/python/image_preprocessing/face_manager.py

```diff
@@ -25,6 +25,30 @@ def __init__(self, shape_predictor_path):
         """
         self.shape_predictor_path = shape_predictor_path
 
+    def face_likeness_extraction(self, image):
+        """
+        This function find a face in the image passed and is optimised
+        to align the face before being returned.
+        Author(s):
+            Stephan Nell
+        Args:
+            image (:obj:'OpenCV image'): Image containing the face we need to detect and extract.
+        Raises:
+            ValueError: If no face can be detected.
+        Returns:
+            obj:'OpenCV image': Any background and unnecessary components are removed and only
+            the aligned face is returned in gray scale for facial likeness.
+        Todo:
+            Return error if no face detected
+
+        """
+        rectangle = self.detect(image)
+        predictor = dlib.shape_predictor(self.shape_predictor_path)
+        face_aligner = FaceAligner(predictor, desiredFaceWidth=256)
+        face_aligned = face_aligner.align(image, image, rectangle)
+        gray_face = cv2.cvtColor(face_aligned, cv2.COLOR_BGR2GRAY)
```

Comment:
The function never intended to be used for display purposes, but solely for facial comparisons and detection which all require a grey image @Nicvaniek and I decided to add the function to clean up repeated functions calls in controllers.py function you mention below.

So we can . . . 
1. Keep the function as it is.
2. Update the already existing extract_face function in this file to return an additional grey image in the already existing tuple it is returning.
3. Or remove this function entirely extract the face with the extract_face function and convert the face to grey as you mention down below.

The final call is up to you but I am for option 1 mainly because I prefer keeping the functionally of Face likeness and text extraction separate.